docs/hello.rst

```diff
@@ -96,23 +95,23 @@ Taichi code can run on CPUs or GPUs. Initialize Taichi according to your hardwar
 
     (OK: supported; N/A: not available)
 
-    With ``arch=ti.gpu``, Taichi will try to run on CUDA.
-    If CUDA is not supported on your machine, Taichi will fall back to Metal or OpenGL.
-    If no GPU backend (CUDA, Metal, or OpenGL) is supported, Taichi will fall back to CPUs.
+    With ``arch=ti.gpu``, Taichi will try to run with CUDA.
+    If CUDA is not supported on your machine, Taichi will fall back on Metal or OpenGL.
```

Comment:
fall back on? Not sure if this is correct phase...sorry.

Reply:
Well, I'm pretty sure. If you google 'fall back to' you will get 'fall back on', like this 
![image](https://user-images.githubusercontent.com/7610945/81803633-146e4900-956c-11ea-87c2-e62094d6dbba.png)

Reply:
Cool! Aplogize for my poor English..

Reply:
No worries! I'm glad I'm helpful 😃